core/client/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -241,23 +241,29 @@ URIStatus getStatus(AlluxioURI path, GetStatusOptions options)
    * Convenience method for {@link #loadMetadata(AlluxioURI, LoadMetadataOptions)} with default
    * options.
    *
+   * @deprecated This is deprecated in version 1.1 and will be removed in version 2.0.
```

Comment:
move to the very bottom of the list of tags, no need to separate by an empty line